clang/test/dpct/vector_type.cu

```diff
@@ -3690,3 +3690,25 @@ void test2() {
   if (std::abs(d2.x) >= 0.01) {}
 }
 
+template <typename T> struct C { T a; };
+// CHECK: /*
+// CHECK-NEXT: DPCT1122:{{[0-9]+}}: The element type of vector type in the migrated code may be different from the original code. Please verify whether it will cause redefinition problem when used in template argument.
```

Comment:
```suggestion
// CHECK-NEXT: DPCT1122:{{[0-9]+}}: ${Type} is migrate to ${Type2} in template declare, it may cause template function or class redefinition, please adjust the code.
```

Reply:
fixed